<launch>
    <node pkg="mil_missions" type="mission_server" name="mission_runner">
        <param name="missions_module" value="navigator_missions" />
        <param name="base_mission" value="NaviGatorMission" />
        <rosparam>
        # Time to lower the grinch, seconds
        grinch_lower_time: 20
        # Time to raise grinch, seconds
        grinch_raise_time: 18
        actuator_timing:
          # Time to wait for lock to engage when deploying thruster
          deploy_lock_time: 0.5
          # Time to wait for thruster to lower when deploying thruster
          deploy_wait_time: 3.0
          # Time to enable retract piston before unlocking when deploying a thruster
          deploy_loosen_time: 1.0
          # Time to wait for lock to engage when retracting thruster
          retract_lock_time: 0.5
          # Time to wait thruster to lift before engaging lock
          retract_wait_time: 3.0
          # Time to extend reload piston when reloading launcher
          launcher_reload_extend_time: 2.5
          # Time to retract reload piston when reloading launcher
          launcher_reload_retract_time: 3.5
        </rosparam>
    </node>
</launch>
